### PR TITLE
Homework22: Alina Shautsova

### PR DESCRIPTION
**Pull request by AlinaShautsova**

### CODE REVIEW

File: robot.log
Comment (Stml89) [inline robot.log:1]: do not store a report in the repository

File: logging_module.py
Comment (Stml89) [inline logging_module.py:1]: robot generate it's own report

**Review by Stml89 — APPROVED** [review]

File: logging_module.py
Comment (AlinaShautsova) [reply logging_module.py:1]: Сan you describe it a little more?

Comment (Stml89) [reply logging_module.py:1]: https://robotframework.org/RobotDemo/#generated-results

**Pull request merged**